scripts/helmcharts/vars_template.yaml

```diff
@@ -16,7 +16,11 @@ postgresql: &postgres
   #     memory: 3000Mi
   #     cpu: 2
 
-clickhouse: {}
+clickhouse:
```

Comment:
This file is deprecated and deleted in dev:HEAD so, not have to make any changes.